appveyor.yml

```diff
@@ -5,7 +5,7 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version
   - npm cache clear
-  - npm i npm@latest -g
+  - npm i npm@5 -g
```

Comment:
Why 5 instead of 6 in this PR? That's what I've meant. It's basically a downgrade from 6 to 5. Or am I wrong? =)